### PR TITLE
LPS-189871 JavaClassParser parseJavaClass() method can't manipulate \r

### PR DESCRIPTION
**Pull request by michaelmdelcavalcanti**
Ticket: [LPS-189871](https://liferay.atlassian.net/browse/LPS-189871)

### CODE REVIEW

**Comment by michaelmdelcavalcanti** [comment]
ci:test:sf

**Comment by michaelmdelcavalcanti** [comment]
ci:test:relevant

**Comment by kevhlee** [comment]
Hey @michaelmdelcavalcanti, looks like there are issues with this change. Some of the SF checks are failing. Can you take a look?

I think the issue is the `else if`. You might need to separate the conditional blocks into multiple `if`'s.

**Review by kevhlee — CHANGES_REQUESTED** [review]
Hey @michaelmdelcavalcanti, looks like there are issues with this change. Some of the SF checks are failing. Can you take a look?

File: modules/util/source-formatter/src/main/java/com/liferay/source/formatter/processor/BaseSourceProcessor.java [hunk 1]
```diff
@@ -287,18 +287,22 @@ protected String format(
...
 			_sourceFormatterArgs.getCheckCategoryNames();
 
-		if (checkCategoryNames.contains("Styling") ||
-			(checkCategoryNames.isEmpty() &&
-			 ListUtil.isEmpty(_sourceFormatterArgs.getCheckNames()))) {
+		if (checkCategoryNames.contains("Upgrade")) {
+			newContent = StringUtil.replace(
+				newContent, StringPool.RETURN_NEW_LINE, StringPool.NEW_LINE);
+		}
+		else if (checkCategoryNames.contains("Styling") ||
```
[the remark this answers is not in the document]
Comment (michaelmdelcavalcanti) [reply modules/util/source-formatter/src/main/java/com/liferay/source/formatter/processor/BaseSourceProcessor.java:294]: Ok, thx.

**Comment by kevhlee** [comment]
ci:forward

**Pull request closed without merging**